app/blogs/[blog]/page.tsx

```
@@ -24,6 +26,9 @@ export default async function BlogPage({ params }: { params: Params }) {
 
 	return (
 		<main className='container mx-auto py-10 px-6'>
+			<div>
+				<a href='/blogs' className='text-blue-500 hover:underline'><FaArrowLeft className={styles.arrowIcon} /></a>
```

ESLint

[error] Do not use an `<a>` element to navigate to `/blogs/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)